controllers/cloud.redhat.com/providers/cronjob/default.go

```diff
@@ -27,7 +27,7 @@ func (j *cronjobProvider) Provide(app *crd.ClowdApp) error {
 
 	for _, cronjob := range app.Spec.Jobs {
 		innerCronjob := cronjob
-		if innerCronjob.Schedule != "" {
+		if innerCronjob.Schedule != "" && !cronjob.Disabled {
```

Comment:
Just curious -- I do not know the reasoning behind the variable re-assignment to start with -- but is there a reason you chose to check `cronjob.Disabled` instead of `innerCronjob.Disabled` here?